velox/connectors/hive/iceberg/tests/IcebergReadTest.cpp

```diff
@@ -1379,6 +1379,465 @@ TEST_F(HiveIcebergTest, equalityDeletesMultipleFiles) {
+    writeToFile(stringDeleteFilePath->getPath(), stringDeleteVector);
+
+    std::vector<IcebergDeleteFile> deleteFiles;
+    deleteFiles.push_back(IcebergDeleteFile(
+        FileContent::kEqualityDeletes,
+        intDeleteFilePath->getPath(),
+        fileFormat_,
+        10,
+        testing::internal::GetFileSize(
+            std::fopen(intDeleteFilePath->getPath().c_str(), "r")),
+        {1}));
+    deleteFiles.push_back(IcebergDeleteFile(
+        FileContent::kEqualityDeletes,
+        stringDeleteFilePath->getPath(),
+        fileFormat_,
+        10,
+        testing::internal::GetFileSize(
+            std::fopen(stringDeleteFilePath->getPath().c_str(), "r")),
+        {2}));
+
+    auto dataFilePath = TempFilePath::create();
+    writeToFile(dataFilePath->getPath(), dataVectors);
+    auto icebergSplits =
+        makeIcebergSplits(dataFilePath->getPath(), deleteFiles);
+
+    createDuckDbTable(dataVectors);
+    std::string duckDbSql = "SELECT * FROM tmp WHERE 1 = 0";
+    assertEqualityDeletes(
+        icebergSplits.back(), asRowType(dataVectors[0]->type()), duckDbSql);
+  }
+
+  // Test 4: Delete none
+  {
+    std::unordered_map<int8_t, std::vector<std::vector<int64_t>>> intDeleteMap;
+    std::unordered_map<int8_t, std::vector<std::vector<StringView>>>
+        stringDeleteMap;
+
+    intDeleteMap.insert({0, {{}}});
+    stringDeleteMap.insert({1, {{}}});
+
+    auto intDeleteFilePath = TempFilePath::create();
+    RowVectorPtr intDeleteVector =
+        makeRowVector({"c0"}, {makeFlatVector<int64_t>(intDeleteMap.at(0)[0])});
+    writeToFile(intDeleteFilePath->getPath(), intDeleteVector);
+
+    auto stringDeleteFilePath = TempFilePath::create();
+    RowVectorPtr stringDeleteVector = makeRowVector(
+        {"c1"}, {makeFlatVector<StringView>(stringDeleteMap.at(1)[0])});
+    writeToFile(stringDeleteFilePath->getPath(), stringDeleteVector);
+
+    std::vector<IcebergDeleteFile> deleteFiles;
+    deleteFiles.push_back(IcebergDeleteFile(
+        FileContent::kEqualityDeletes,
+        intDeleteFilePath->getPath(),
+        fileFormat_,
+        0,
+        testing::internal::GetFileSize(
+            std::fopen(intDeleteFilePath->getPath().c_str(), "r")),
+        {1}));
+    deleteFiles.push_back(IcebergDeleteFile(
+        FileContent::kEqualityDeletes,
+        stringDeleteFilePath->getPath(),
+        fileFormat_,
+        0,
+        testing::internal::GetFileSize(
+            std::fopen(stringDeleteFilePath->getPath().c_str(), "r")),
+        {2}));
+
+    auto dataFilePath = TempFilePath::create();
+    writeToFile(dataFilePath->getPath(), dataVectors);
+    auto icebergSplits =
+        makeIcebergSplits(dataFilePath->getPath(), deleteFiles);
+
+    createDuckDbTable(dataVectors);
+    std::string duckDbSql = "SELECT * FROM tmp";
+    assertEqualityDeletes(
+        icebergSplits.back(), asRowType(dataVectors[0]->type()), duckDbSql);
+  }
+}
+
+TEST_F(HiveIcebergTest, equalityDeletesMixedTypesTinyintVarbinary) {
```

Comment:
Why did you add it here? The logics are essentially the same as HiveIcebergEqualityDeletesTest::testSingleColumnEqualityDeletes